### PR TITLE
Rf rename chained block #1320

### PR DESCRIPTION
**Pull request by kogot**

### CODE REVIEW

File: src/Stratis.Bitcoin.Features.Miner/PowBlockAssembler.cs [hunk 1]
```diff
@@ -17,15 +17,13 @@ public abstract class BlockAssembler
...
+        /// <param name="chainTip">Tip of the chain that this instance will work with without touching any shared chain resources.</param>
         /// <returns>The contructed <see cref="BlockTemplate"/>.</returns>
-        public abstract BlockTemplate Build(ChainedBlock chainTip, Script scriptPubKey);
-
-        /// <summary>
-        /// Tip of the chain that this instance will work with without touching any shared chain resources.
-        /// </summary>
-        protected ChainedBlock ChainTip { get; set; }
+        public abstract BlockTemplate Build(ChainedHeader chainTip, Script scriptPubKey);
+        /// <summary>Tip of the chain that this instance will work with without touching any shared chain resources.</summary>
```
Comment (bokobza) [inline src/Stratis.Bitcoin.Features.Miner/PowBlockAssembler.cs]: Can you add a new line above the ChainTip method please?

**Comment by dangershony** [comment]
@kogot ops... please merge.

**Comment by kogot** [comment]
@dangershony @bokobza all conflicts resolved

**Comment by dangershony** [comment]
ah please fix the conflict again.
change looks good to me its only a name change

**Comment by kogot** [comment]
@dangershony it's ready now

**Comment by dangershony** [comment]
Please fix conflicts again, you are next to go in so we finish this.

**Pull request merged**